packages/react-components/react-drawer/stories/src/Drawer/DrawerResizable.stories.tsx

```diff
@@ -168,15 +193,33 @@ export const Resizable = () => {
                 <DialogContent>
                   <div className={styles.dialogContent}>
                     <Label htmlFor={inputId}>Enter desired drawer width pixels:</Label>
-                    <Input id={inputId} onChange={onResizeInputChange} defaultValue={resizeInput} type="number" />
+                    <Input
+                      id={inputId}
+                      ref={inputRef}
+                      onChange={onResizeInputChange}
+                      defaultValue={resizeInput}
+                      type="number"
+                      className={showMessage ? styles.invalidInput : ''}
+                    />
+                    {showMessage ? (
+                      <Label className={styles.errorMessage}>
+                        Recommended minimum width of the drawer should be greater than or equal to `240px`.
+                      </Label>
+                    ) : null}
                   </div>
                 </DialogContent>
                 <DialogActions>
                   <DialogTrigger disableButtonEnhancement>
                     <Button
                       appearance="primary"
-                      onClick={() => {
-                        setSidebarWidth(Number(resizeInput));
+                      onClick={e => {
```

Comment:
one other thought -- it'd be good to demo adding arrow key resizing behavior as well, when focus is on this button. Left/right arrows would resize by some chosen fixed amount (could be a static number of pixels, or you could internally calculate something like 10% of the total width if you wanted to be fancy)

Reply:
This is a great idea! I thought about something similar. I'll ask Leslie and Gouttierre. I think it would be great to add it

Reply:
Added resize with arrows :)